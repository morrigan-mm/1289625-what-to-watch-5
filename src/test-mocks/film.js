const film = {
  id: 15,
  poster: `#poster`,
  preview: `#preview`,
  video: `#video`,
  cover: `#cover`,
  title: `The Revenant`,
  genre: `Action`,
  releaseDate: 2015,
  description: `A frontiersman on a fur trading`,
  rating: 4,
  ratingCount: 618498,
  director: `The Director`,
  actors: [`actor1`, `actor2`],
  runtime: 156,
  addedToMyList: false,
  backgroundColor: `#92918B`,
  backgroundImage: `#bgimage`
};

export default film;
