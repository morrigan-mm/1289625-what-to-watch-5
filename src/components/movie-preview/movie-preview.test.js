import React from "react";
import renderer from "react-test-renderer";
import film from "../../test-mocks/film";
import MoviePreview from "./movie-preview";

const {poster, video} = film;

describe(`MoviePreview`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(
          <MoviePreview poster={poster} video={video} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
