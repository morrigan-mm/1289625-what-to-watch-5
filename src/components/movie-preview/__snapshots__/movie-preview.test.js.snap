// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePreview Should render correctly 1`] = `
<video
  className="small-movie-card__image"
  height="175"
  muted={true}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  poster="#poster"
  preload="none"
  src="#video"
  width="280"
/>
`;
